.ci/scripts/prepare_old_deps.sh

```diff
@@ -69,15 +59,12 @@ with open('pyproject.toml', 'w') as f:
 "
 python3 -c "$REMOVE_DEV_DEPENDENCIES"
 
-pipx install poetry==1.1.14
-~/.local/bin/poetry lock
+pip install poetry==1.2.0
+poetry lock
```

Comment:
FWIW it's possible that poetry's dependencies might conflict with those of `toml` and `wheel` here. It's probably not a problem but calling it out for paranoia's sake.

Reply:
Shouldn't `poetry lock` here be ignoring toml and wheels deps?

Reply:
You're right that the virtualenv managed by poetry will be independent of toml and wheel. What I mean: the depedencies of the poetry package itself, i.e. the stuff listed here

```
$ pipx runpip poetry show poetry
Name: poetry
Version: 1.2.0
Summary: Python dependency management and packaging made easy.
Home-page: https://python-poetry.org/
Author: Sébastien Eustace
Author-email: sebastien@eustace.io
License: MIT
Location: /home/dmr/.local/pipx/venvs/poetry/lib/python3.10/site-packages
Requires: cachecontrol, cachy, cleo, crashtest, dulwich, html5lib, jsonschema, keyring, packaging, pexpect, pkginfo, platformdirs, poetry-core, poetry-plugin-export, requests, requests-toolbelt, shellingham, tomlkit, urllib3, virtualenv
Required-by: poetry-plugin-export
```

might have conflicts with the toml and wheel versions we've just installed.

But a) it doesn't look like there are any conflicts, and b) I think pip would at least warn us about them.

TL;DR ignore this comment, it's all good